File: six_odd_numbers.c
#include <stdio.h>

int main(){
	int num, cont=0;
	scanf("%d", &num);
	do {
		if (num%2 != 0){
			printf("%d\n", num);
			cont++;
		}
		num++;
		if (cont == 6){
			break;
		}
	} while(1);
	return 0;
}

File: Java/SixOddNumbers.java
import java.util.Scanner;
public class SixOddNumbers {
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int num = scan.nextInt();
        int cont=0;
        for (int i=num; cont<6; i++){
            if (i%2!=0){
                System.out.println(i);
                cont++;
            }
        }
    }
}
